### PR TITLE
 ANN: Detect E0518: Inline attribute is allowed only on functions 

### PR DESCRIPTION
**Pull request by mibac138**
Related to #886.

### CODE REVIEW

File: src/main/kotlin/org/rust/ide/annotator/fixes/RemoveAttrFix.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.intellij.codeInspection.LocalQuickFixOnPsiElement
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.ext.RsAttr
+import org.rust.lang.core.psi.ext.name
+
+class RemoveAttrFix(attr: RsAttr) : LocalQuickFixOnPsiElement(attr) {
+    private val _text = "Remove attribute" + when (val name = attr.metaItem.name) {
```
Comment (mchernyavsky) [inline src/main/kotlin/org/rust/ide/annotator/fixes/RemoveAttrFix.kt]: You can use `run` here:
```kotlin
private val _text: String = run {
    val suffix = attr.metaItem.name?.let { " `$it`" } ?: ""
    "Remove attribute$suffix"
}
```
Also you can move this code to `getText`:
```kotlin
override fun getText(): String {
    val attr = startElement as? RsAttr
    val suffix = attr?.metaItem?.name?.let { " `$it`" } ?: ""
    return "Remove attribute$suffix"
}
```

**Comment by mchernyavsky** [comment]
Could you please squash/rebase your commits?

**Comment by mibac138** [comment]
@mchernyavsky done

**Comment by mibac138** [comment]
@mchernyavsky done

**Pull request merged**